### PR TITLE
use &larr; / &rarr; instead of unicode arrows; fix #4702

### LINKED ISSUE #4702: Documentation: request fix the arrows in standard-platform.md
Ref: https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/standard-platform.md

The current unicode character used doesn't render on Chrome on Win 10 (UK English), and possibly a lot of other configurations - making it hard to understand the tables. Fonts are tricky, but there's the handy `&rarr;` that would look almost identical, and would render just about everywhere.

Would be happy to submit a PR if it helps.

### PR DESCRIPTION
**Pull request by mgravell**
(as discussed in related issue)

### CODE REVIEW

**Comment by JonHanna** [comment]
What about maybe using → directly, rather than via an entity reference? The source would then more closely match the rendered version, or is something going awry with encodings?

**Comment by mgravell** [comment]
I'll be honest: I didn't test that. Simply: &larr; and &rarr; are the
idiomatic and most correct ways to express left/right arrows in both html
and markdown. Given that it has already failed once, I was defaulting to
"will obviously work" rather than "might work, not really sure".

**Comment by JonHanna** [comment]
It should work, since the problem in the original seems to be font-related (judging from how it works fine on my own rig with the same OS/Browser combo you reported about, suggesting the difference comes down to our installed fonts), and hence if `→` didn't work, then neither would `&rarr;`. It's just a matter of whether you think what I said about the → being visible as → in the source being worth it or not.

**Comment by stephentoub** [comment]
LGTM

File: Documentation/project-docs/standard-platform.md [hunk 1]
```diff
@@ -292,132 +292,132 @@ Tooling support for the `netstandard` TFM is as follows. This list will be updat
 
 ### Legend 
 - `X` - API appeared in specific .NET Platform Standard version
-- `⇠` - API version determined by nearest `X` e.g. In the table below, if you target .NET Platform Standard version 1.4 and reference Microsoft.CSharp, you'd get the 1.0 API version.
+- `&larr;` - API version determined by nearest `X` e.g. In the table below, if you target .NET Platform Standard version 1.4 and reference Microsoft.CSharp, you'd get the 1.0 API version.
```
Comment (akoeplinger) [inline Documentation/project-docs/standard-platform.md]: the backticks make this one appear as the raw text, probably need to remove the backticks in this case.

Comment (JonHanna) [inline Documentation/project-docs/standard-platform.md]: Another advantage that using that character directly would have.

Comment (akoeplinger) [inline Documentation/project-docs/standard-platform.md]: fwiw, I experienced the unicode arrow problem on one of my boxes too, so this is definitely better :)

I'm not really concerned about the source readability as the markdown isn't formatted for that anyway (look at those tables...).

**Pull request merged**